src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/WorkflowRunController.kt

```diff
@@ -2,13 +2,23 @@ package org.vaccineimpact.orderlyweb.controllers.web
 
 import org.vaccineimpact.orderlyweb.ActionContext
 import org.vaccineimpact.orderlyweb.controllers.Controller
-import org.vaccineimpact.orderlyweb.db.Orderly
-import org.vaccineimpact.orderlyweb.db.OrderlyClient
 import org.vaccineimpact.orderlyweb.viewmodels.WorkflowRunViewModel
+import org.vaccineimpact.orderlyweb.db.repositories.OrderlyWebWorkflowRunRepository
+import org.vaccineimpact.orderlyweb.db.repositories.WorkflowRunRepository
+import org.vaccineimpact.orderlyweb.models.WorkflowRun
 
-class WorkflowRunController(context: ActionContext, val orderly: OrderlyClient) : Controller(context)
+class WorkflowRunController(
+    context: ActionContext,
+    private val workflowRunRepository: WorkflowRunRepository
+) : Controller(context)
 {
-    constructor(context: ActionContext) : this(context, Orderly(context))
+    constructor(context: ActionContext) : this(context, OrderlyWebWorkflowRunRepository())
+
+    fun getRunWorkflowDetails(): WorkflowRun
```

Comment:
```suggestion
    fun getWorkflowRunDetails(): WorkflowRun
```
Let's be consistent and ensure that the JSON endpoints all refer to a `WorkflowRun`. I know this is annoying - but we made the same mistake with reports and it has got very confusing. Sorry, I missed this first time round.

I know We have ``getRunWorkflow` for the Freemarker page below, but that's fine because it's about the "Run workflow" page, rather than retrieving details about workflow runs.